docs/slurm-troubleshooting.md

```diff
@@ -90,6 +91,23 @@ The solution here is to [request more of the specified quota](#gcp-quotas),
 
 [partition-machine-type]: community/modules/compute/schedmd-slurm-gcp-v6-nodeset/README.md#input_machine_type
 
+#### Insufficient Capacity
+
+It may be that the zone the partition is deployed in has no remaining capacity to create the
+compute nodes required to run your submitted job.
+
+You can confirm this by SSHing into the `controller` VM and checking the
+`resume.log` file:
+
+```shell
+$ cat /var/log/slurm/resume.log
+... bulkInsert operation errors: VM_MIN_COUNT_NOT_REACHED ...
```

Comment:
As mentioned elsewhere, this is one possible error. Maybe say to check resume.log for possible errors, and then add as a NOTE that `VM_MIN_COUNT_NOT_REACHED` can be seen due to insufficient capacity. Basically make the statement less explicit. I don't want someone seeing this error and thinking it must be capacity. On the other hand, they might see a different error to this one. It is still good to call out `VM_MIN_COUNT_NOT_REACHED` as this particular message is not very intuitive.

Reply:
Makes sense, updated.